### PR TITLE
New CBA Tool Implementation

### PR DESCRIPTION
**Pull request by avpeery**
This PR implements the new CBA Tool, and produces uncontrolled and controlled values for script cba algorithm tables, so there is now a drop down to choose uncontrolled versus controlled view on the CBA page.

There are two versions of the CBA Tool. The one committed by Robbie are in the files under ec2setup/CostBenefitAnalysis/ .. I made some changes in order to read the data files from the S3 bucket - the changed version being used in the project is under webserver/CostBenefitAnalysis. The CBA runner takes about 3 minutes to complete. 

I had to make an update to the models, as mentioned, to accommodate uncontrolled and controlled values, instead of just values.

Screen Shot 2020-10-28 at 10 44 28 PM
Screen Shot 2020-10-28 at 10 44 22 PM

### CODE REVIEW

File: webserver/script/CostBenefitAnalysis/python_code/loadprofile_class.py [hunk 1]
```diff
@@ -207,14 +207,15 @@ def add_loadprofiles(name, loadprofile1, loadprofile2, timesteps, model_years, v
 
         return new_loadprofile
 
-def generate_annual_stream_from_load(load_profile, energy_marginal_cost, capacity_marginal_cost,
+def generate_annual_stream_from_load(load_profile, energy_marginal_cost,  capacity_marginal_cost,
```
Comment (derins) [inline webserver/script/CostBenefitAnalysis/python_code/loadprofile_class.py]: extra space here

File: webserver/script/tasks.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 @shared_task
 def run_cba_tool(county_data, profile_name):
-    split_file(county = county_data)
-    ModelInstance()
-    UploadToPostgres(load_profile = profile_name)
-    return
+    split_file(county = county_data[0], controlled_types = ["uncontrolled", "e19controlled"])
```
Comment (derins) [inline webserver/script/tasks.py:13]: inconsistent quotes here

**Review by derins — APPROVED** [review]
Everything else seems good

File: ec2setup/algorithms/CostBenefitAnalysis/python_code/billcalculator_class.py [hunk 3]
```diff
@@ -187,7 +187,9 @@ def calculate_bill(self, model_years, weekday_weekend_count,workplace_peak_hour,
...
                                 + total_monthly_max_charge \
-                                + total_onpeak_max_charge \
-                                + total_partpeak_max_charge1 \
-                                + total_partpeak_max_charge2 \
-                                + fixed_monthly_charges + fixed_daily_charges
\ No newline at end of file
+                                + fixed_monthly_charges + fixed_daily_charges
+                                # + total_onpeak_max_charge \
+                                # + total_partpeak_max_charge1 \
+                                # + total_partpeak_max_charge2 \
```
Comment (jongoncalves) [inline ec2setup/algorithms/CostBenefitAnalysis/python_code/billcalculator_class.py:193]: Can we get rid of these comments?

**Review by jongoncalves — APPROVED** [review]

**Pull request merged**